api/src/main/kotlin/eu/codeloop/thehub/properties/JobProperties.kt

```diff
@@ -0,0 +1,8 @@
+package eu.codeloop.thehub.properties
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+
+@ConfigurationProperties(prefix = "thehub.job")
+class JobProperties {
+    lateinit var `approved-at-default`: String
```

Comment:
- [ ] Please remove `JobProperties` - we should use `DefaultValues` in whole code